### ISSUE TITLE
[FSM][P2] Accessibility issue setting or removing label for underlying button on fd-shellbar-action

### ISSUE THREAD

**Issue by VGio** [SAP/fundamental-ngx#13791]
### Describe your proposal

Dear fundamentals-ngx Team,
we had accessibility tests for our UI and some issues came up which we cannot set in our code.
We use the Shellbar or the Sidebar with those actions with aria label and label like here https://sap.github.io/fundamental-ngx/#/core/shellbar
Now the issue is the underlying button which get created do not have those label or aria label. 
This is a problem from accessibility stand point.
Is there a way to pass it to the buttons ? Or for use to add those in any way ?

### Which versions of Angular and Fundamental Library for Angular are affected?

Angular : 19 , fundamental-ngx 0.56

### Steps to Reproduce (for bugs)

Add label to any shellbar-action and look into the code to see button which gets created not having those 

### Priority

None

### Declaration

- [x] I'm not disclosing any internal or sensitive information.

**Comment by InnaAtanasova** [comment]
Hi @VGio , thanks for reporting the issue. I provided a fix in this PR: https://github.com/SAP/fundamental-ngx/pull/13795
Could you check the updated documentation examples and let us know if this is enough to address the issue on your side?
https://deploy-preview-13795--fundamental-ngx.netlify.app/#/core/shellbar

Now you can add to the actions objects additionally an ariaLabel and title
<img width="540" height="399" alt="Image" src="https://github.com/user-attachments/assets/cf95b1f8-cff6-428f-b736-224498928c1f" />

this will set a title for the button:

<img width="279" height="147" alt="Image" src="https://github.com/user-attachments/assets/f4be83b6-59d8-4558-b3c7-29b18976db20" />

as well as an aria-label
<img width="1311" height="711" alt="Image" src="https://github.com/user-attachments/assets/40edc07b-769b-4922-b619-6ac25e18cb77" />

Let us know if we missed something. Thank you!

**Comment by VGio** [comment]
Hey @InnaAtanasova , @droshev or Ngx Team will this be also downported to the LTS version since we are using version 0.56 ngx-fundamentals ?

**Comment by droshev** [comment]
downported to 
- [0.58.2](https://github.com/SAP/fundamental-ngx/releases/tag/v0.58.2)
- [0.56.9](https://github.com/SAP/fundamental-ngx/releases/tag/v0.56.9)
